larpandoracontent/LArThreeDReco/LArThreeDBase/TwoViewMatchingControl.h

```diff
@@ -73,6 +73,7 @@ class TwoViewMatchingControl : public NViewMatchingControl
 
     template <typename U>
     friend class NViewMatchingAlgorithm;
+    friend class TwoViewDeltaRayMatchingAlgorithm;    
```

Comment:
As far as I can tell, the only reason for this class to be a friend is to have access to the private member variable m_hitTypeToIndexMap. If that's correct, I think it's preferable to define a public function in this class that can take a request for an index based on a view (returning something like -1 if the view isn't found in the map), rather than giving another class access to all of the implementation details of this class for the sake of a single variable.